README.adoc

```diff
@@ -1,8 +1,8 @@
 = Address Book (Level 4)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
+https://travis-ci.org/KevinChuangCH/main[image:https://travis-ci.org/KevinChuangCH/main.svg?branch=master[Build Status]]
```

Comment:
Links should be changed on the branch that updates the Developer Guide, and into our organization's main repo.